src/Okta.Idx.Sdk/IIdxClient.cs

```diff
@@ -44,16 +44,18 @@ public interface IIdxClient : IOktaClient
         /// </summary>
         /// <param name="authenticationOptions">The authentication options.</param>
         /// <param name="cancellationToken">The cancellation token.</param>
+        /// <param name="recoveryToken">The recovery token.</param>
         /// <returns>The authentication response.</returns>
-        Task<AuthenticationResponse> AuthenticateAsync(AuthenticationOptions authenticationOptions, CancellationToken cancellationToken = default);
+        Task<AuthenticationResponse> AuthenticateAsync(AuthenticationOptions authenticationOptions, CancellationToken cancellationToken = default, string recoveryToken = null);
 
         /// <summary>
         /// Initiates the password recovery flow
         /// </summary>
         /// <param name="recoverPasswordOptions">The password recovery options.</param>
         /// <param name="cancellationToken">The cancellation token.</param>
+        /// <param name="recoveryToken">The recovery token.</param>
         /// <returns>The authentication response.</returns>
-        Task<AuthenticationResponse> RecoverPasswordAsync(RecoverPasswordOptions recoverPasswordOptions, CancellationToken cancellationToken = default);
+        Task<AuthenticationResponse> RecoverPasswordAsync(RecoverPasswordOptions recoverPasswordOptions, CancellationToken cancellationToken = default, string recoveryToken = null);
```

Comment:
In order to be consistent with the existing pattern, recoveryToken should be an optional property of RecoverPasswordOptions.